tests/unit/auth/v2/headerAuthCheck.js

```diff
@@ -0,0 +1,40 @@
+const assert = require('assert');
+
+const headerAuthCheck =
+    require('../../../../lib/auth/v2/headerAuthCheck').check;
+const DummyRequestLogger = require('../../helpers').DummyRequestLogger;
+
+const log = new DummyRequestLogger();
+
+describe('v2: headerAuthCheck', () => {
+    it('should not fail if the token is missing', () => {
```

Comment:
can you factorize here?

```js
// 0 -> assert.notStrictEqual
// 1 -> assert.StrictEqual
const tests = [
    ['should do X', {}, 'strictEqual'],
    ['should not fail if the token is missing', {}, 'notStrictEqual'],
]

tests.forEach(test => {
    // your test
    assert[test[2]](x, true);
});
```
You can replace array by object for making it more readable